Fixes the following warnings:

getopt.c: In function '_getopt_initialize':
getopt.c:382: warning: unused parameter 'argc'
getopt.c:382: warning: unused parameter 'argv'
getopt.c: In function '_getopt_internal':
getopt.c:681: warning: suggest explicit braces to avoid ambiguous 'else'

Index: getopt.c
===================================================================
RCS file: /sources/make/make/getopt.c,v
retrieving revision 1.22
diff -u -p -r1.22 getopt.c
--- make-3.82.orig/src/getopt.c	25 Oct 2009 18:56:45 -0000	1.22
+++ make-3.82/src/getopt.c	5 Jul 2010 20:05:22 -0000
@@ -436,6 +436,9 @@ _getopt_initialize (int argc, char *cons
     }
   else
     nonoption_flags_len = 0;
+#else
+  (void)argc;
+  (void)argv;
 #endif
 
   return optstring;
